src/ign.cc

```diff
@@ -977,3 +977,42 @@ extern "C" IGNITION_FUEL_TOOLS_VISIBLE int editUrl(
 
   return 1;
 }
+
+extern "C" IGNITION_FUEL_TOOLS_VISIBLE int update(
+    const char *_onlyModels, const char *_onlyWorlds)
```

Comment:
Add `header` support here, which should allow updating private models and worlds.

Reply:
Added in 64622f2